platforms/software/dependency-management/src/integTest/groovy/org/gradle/integtests/resolve/transform/ArtifactTransformIntegrationTest.groovy

```diff
@@ -1385,46 +1393,38 @@ Found the following transforms:
         fails "resolve"
 
         then:
-        failure.assertHasCause """Found multiple transforms that can produce a variant of project :lib with requested attributes:
+        failure.assertHasCause """Found multiple transformation chains that produce a variant of 'project :lib' with requested attributes:
   - artifactType 'transformed'
   - usage 'api'
-Found the following transforms:
-  - From 'configuration ':lib:compile' variant variant1':
+Found the following transformation chains:
+  - From configuration ':lib:compile' variant 'variant1':
       - With source attributes:
           - artifactType 'jar'
           - buildType 'release'
           - flavor 'free'
           - usage 'api'
-      - Candidate transform(s):
-          - Transform 'BrokenTransform' producing attributes:
-              - artifactType 'transformed'
-              - buildType 'release'
-              - flavor 'free'
-              - usage 'api'
-  - From 'configuration ':lib:compile' variant variant2':
```

Comment:
What happened to `variant2` in the new error message? Unless `free` and `paid` are compatible, I would expect these to be distinct sets of final attributes and the resulting source + chain should be reported